sr_grasp_stability/src/sr_grasp_stability/polygon_computation.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python
+
+import numpy as np
+
+# class Polygon:
```

Comment:
Should this be a class or just have methods? The difference is you have to instantiate an object, which isn't necessary in this case.